### DISCUSSION TITLE
What can be happening?

### DISCUSSION THREAD

**Discussion by camoralb** [nutgram/nutgram#541] [General]
I install it via composer with PHP 8.2

Fatal error: Uncaught Error: Undefined constant "SergiX44\Nutgram\RunningMode\STDERR" in /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Container.php:151 Stack trace: #0 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Container.php(151): ReflectionClass->newInstance() #1 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Container.php(51): SergiX44\Container\Container->resolve('SergiX44\\Nutgra...') #2 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Definition.php(48): SergiX44\Container\Container->get('SergiX44\\Nutgra...') #3 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Container.php(43): SergiX44\Container\Definition->make(Object(SergiX44\Container\Container)) #4 /Users/carlosmorales/Downloads/bot/vendor/nutgram/nutgram/src/Nutgram.php(200): SergiX44\Container\Container->get('SergiX44\\Nutgra...') #5 /Users/carlosmorales/Downloads/bot/test.php(25): SergiX44\Nutgram\Nutgram->run() #6 {main} Next SergiX44\Container\Exception\NotFoundException: Cannot resolve 'SergiX44\Nutgram\RunningMode\Polling' in /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Exception/NotFoundException.php:13 Stack trace: #0 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Container.php(53): SergiX44\Container\Exception\NotFoundException::notResolvable('SergiX44\\Nutgra...', Object(Error)) #1 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Definition.php(48): SergiX44\Container\Container->get('SergiX44\\Nutgra...') #2 /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Container.php(43): SergiX44\Container\Definition->make(Object(SergiX44\Container\Container)) #3 /Users/carlosmorales/Downloads/bot/vendor/nutgram/nutgram/src/Nutgram.php(200): SergiX44\Container\Container->get('SergiX44\\Nutgra...') #4 /Users/carlosmorales/Downloads/bot/test.php(25): SergiX44\Nutgram\Nutgram->run() #5 {main} thrown in /Users/carlosmorales/Downloads/bot/vendor/sergix44/container/src/Exception/NotFoundException.php on line 13

**Comment by Lukasss93** [comment]
please open an issue with more informations
